pyccel/stdlib/numpy/numpy_c.c

```diff
@@ -17,8 +17,10 @@ double  fsign(double x)
     return SIGN(x);
 }
 
+#ifndef __NVCC__
 /* numpy.sign for complex */
 double complex csign(double complex x)
 {
     return x ? ((!creal(x) && cimag(x) < 0) || (creal(x) < 0) ? -1 : 1) : 0;
 }
+#endif
```

Comment:
Missing end of line

Reply:
[9da03c4](https://github.com/pyccel/pyccel-cuda/pull/32/commits/9da03c4dcabf0e13a61890b17738f945ae09bf32)